File: ruby/vristo-ruby-starter/app/controllers/apps_controller.rb
class AppsController < ApplicationController
    def chat
       render layout: "default", :template => 'apps/chat'
    end
    def mailbox
        render layout: "default", :template => 'apps/mailbox'
     end
     def todolist
        render layout: "default", :template => 'apps/todolist'
     end
     def notes
        render layout: "default", :template => 'apps/notes'
     end
     def contacts
        render layout: "default", :template => 'apps/contacts'
     end
     def calendar
        render layout: "default", :template => 'apps/calendar'
     end
     def scrumboard
        render layout: "default", :template => 'apps/scrumboard'
     end
     def invoice_add
        render layout: "default", :template => 'apps/invoice/add'
     end
     def invoice_edit
        render layout: "default", :template => 'apps/invoice/edit'
     end
     def invoice_list
        render layout: "default", :template => 'apps/invoice/list'
     end
     def invoice_preview
        render layout: "default", :template => 'apps/invoice/preview'
     end
  
end

File: symfony/vristo-symfony-starter/src/Controller/AppsController.php
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class AppsController extends AbstractController
{
    public function chat(): Response
    {
        return $this->render('apps/chat.html.twig');
    }

    public function mailbox(): Response
    {
        return $this->render('apps/mailbox.html.twig');
    }

    public function todo_list(): Response
    {
        return $this->render('apps/todolist.html.twig');
    }

    public function notes(): Response
    {
        return $this->render('apps/notes.html.twig');
    }

    public function scrumboard(): Response
    {
        return $this->render('apps/scrumboard.html.twig');
    }

    public function contacts(): Response
    {
        return $this->render('apps/contacts.html.twig');
    }

    public function invoice_list(): Response
    {
        return $this->render('apps/invoice/list.html.twig');
    }

    public function invoice_add(): Response
    {
        return $this->render('apps/invoice/add.html.twig');
    }

    public function invoice_preview(): Response
    {
        return $this->render('apps/invoice/preview.html.twig');
    }

    public function invoice_edit(): Response
    {
        return $this->render('apps/invoice/edit.html.twig');
    }

    public function calendar(): Response
    {
        return $this->render('apps/calendar.html.twig');
    }
}
